src/app/communities/communityUtils.tsx

```diff
@@ -144,12 +144,25 @@ export const vanguardCommunity = {
   ),
 }
 
+export const betaBuilders: CommunityItem = {
+  leftImageSrc: '/images/nfts/bb-thumb.png',
+  title: 'Beta Builders',
+  subtitle: 'DeFi',
+  description:
+    'The Beta Builders collection is a badge of honor for the visionaries shaping the future of Bitcoin through RootstockCollective. It celebrates the pioneering Builders, Developers, Protocols, and dApps whose innovations are driving Rootstock in becoming the leading Bitcoin Layer 2.',
```

Comment:
there is one more piece you need to add here for Toby that is not in his word document but in the ticket.

```
SPECIAL POWERS: Voting Booster
REQUIREMENT: First 50 CollectiveRewards Builders, Air-Drop
ACTIVATION: Apr 2025
```